use-cases/manual-compliance.md

```diff
@@ -1,31 +1,45 @@
-# Manually onboard an AWS account for CSPM
+# Manually Onboard an AWS Account for CSPM
 
-Please note: This method of installation will **only** support CSPM (Compliance). 
-The following features **will not work**: Threat Detection, Identity and Access, Image Scanning. 
-To install other features, please follow our full [Installation Guide](https://docs.sysdig.com/en/docs/installation/sysdig-secure/connect-cloud-accounts/aws/)
+To enable CSPM (Compliance) in your AWS account, you create the following resources on the Sysdig Secure SaaS backend:
 
-In each AWS account where you would like to run CSPM, we must create an IAM Role with `SecurityAudit` permissions that Sysdig is able to assume.
+    - An `account` representing the AWS account for which you want to enable CSPM
+    - A trust-relationship `component` that represents the IAM Role in your AWS account
+    - A CSPM feature that indicates CSPM scans should be run against this account
```

Comment:
can we keep "feature" as a code block for consistency?

Reply:
done